nemo_rl/utils/logger.py

```diff
@@ -430,6 +461,101 @@ def log_histogram(self, histogram: list[Any], step: int, name: str) -> None:
         return
 
 
+class OneLogger(LoggerInterface):
+    """OneLogger backend."""
+
+    def __init__(self, cfg: OneLoggerConfigDict, log_dir: Optional[str] = None):
+        if not HAVE_ONE_LOGGER:
+            print("Warning: nv_one_logger not installed. OneLogger will be disabled.")
+            return
+
+        self.provider = OneLoggerProvider.instance()
+        project = cfg.get("project", "nemo_rl")
+        name = cfg.get("name", "nemo_rl_run")
```

Comment:
these default should come from yaml

```suggestion
        project = cfg["project"]
        name = cfg["name"]
```